dstu3/src/main/resources/logback.xml

```diff
@@ -37,4 +37,9 @@
         <!-- <appender-ref ref="htmlAppender" /> -->
     </root>
 
+    <logger name = "org.opencds.cqf.dstu3" level="DEBUG" additivity="false">
```

Comment:
We're targeting CQL engine logging specifically, so this name should probably be:
`org.opencds.cqf.cql.engine.debug`